modules/payloads/singles/windows/pingback_reverse_tcp.rb

```diff
@@ -22,15 +21,14 @@ module MetasploitModule
 
   def initialize(info = {})
     super(merge_info(info,
-      'Name'          => 'Windows x86 Pingback, Reverse TCP Inline',
-      'Description'   => 'Connect back to attacker and report UUID (Windows x86)',
-      'Author'        => [ 'bwatters-r7' ],
-      'License'       => MSF_LICENSE,
-      'Platform'      => 'win',
-      'Arch'          => ARCH_X86,
-      'Handler'       => Msf::Handler::ReverseTcp,
-      'Session'       => Msf::Sessions::Pingback
-    ))
+                     'Name' => 'Windows x86 Pingback, Reverse TCP Inline',
```

Comment:
What happened here?

Reply:
Looks like you accounted for only `update_info` in your rule.

Reply:
Thanks! Will fix 👀